pkg/crc/ssh/ssh.go

```diff
@@ -28,6 +28,7 @@ func (runner *SSHRunner) Run(command string) (string, error) {
 }
 
 func (runner *SSHRunner) SetTextContentAsRoot(destFilename string, content string, mode os.FileMode) error {
+	logging.Debugf("Creating %s with permissions 0%o in the CRC VM", destFilename, mode)
```

Comment:
This should be part of earlier commit I think.

Reply:
yes!